### DISCUSSION TITLE
Some apps using weblate for localization can't use Chinese translation

### DISCUSSION THREAD

**Discussion by lumingzh** [WeblateOrg/weblate#7386] [Q&A] [answered]
I'm not sure it is weblate's bug or app's bug. The apps I know are [Bottles](https://github.com/bottlesdevs/Bottles/issues/574), [Junction](https://github.com/sonnyp/Junction/issues/61), [Collision](https://github.com/GeopJr/Collision/issues/58), all not work.

OS: fedora silverblue 35
All apps installed from flathub. And I think maybe it's relate to [this](https://github.com/WeblateOrg/weblate/discussions/5666), not sure.

**Comment by nijel** [comment, marked as answer]
They might need to adjust [Language code style](https://docs.weblate.org/en/latest/admin/projects.html#language-code-style), there is no default that would fit everybody, see [Adding new translations](https://docs.weblate.org/en/latest/admin/languages.html#adding-new-translations) for more info.

Or they can just rename the files in the repository, Weblate will pick them up correctly.

**Reply by lumingzh** [reply]
Yes, zh_CN works.
